pkg/sql/compile/view.go

```diff
@@ -72,8 +75,9 @@ func constructView(bat *batch.Batch, vars []string) {
 		if n > UnitLimit {
 			n = UnitLimit
 		}
+		// zValues is used to flag null, in sql null is not equal to any value (including null), so special flagging is needed.
 		copy(zValues[:n], OneInt64s[:n])
-		for j, vec := range vecs {
+		for j, vec := range vecs { // // combine multiple attributes into a single key
```

Comment:
You have about 20 warnings in this file, 
`for i, _ := range xxx`
Go warns you that _ is not necessary.   Please fix.

Reply:
> > Add comments and test cases for the times operator, and give an example for testing the operator.Note that the current comments and test coverage is not very good, and I will add comments and more comprehensive test cases for all the operators next. (If the test methods in this pr are considered to be correct)
> 
> Since quite a lot of code was due to performance considerations, the test.go file should include a few BenchmarkXXX method to justify the complexity.

ok